otelcollector/deploy/addon-chart/ccp-metrics-plugin/templates/ama-metrics-deployment.yaml

```diff
@@ -0,0 +1,233 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ama-metrics-ccp
+  namespace: {{ .Values.global.commonGlobals.Customer.Namespace }}
+  labels:
+    component: ama-metrics-ccp
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  paused: false
+  selector:
+    matchLabels:
+      rsName: ama-metrics-ccp
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        rsName: ama-metrics-ccp
+      annotations:
+        agentVersion: "0.0.0.1"
```

Comment:
same versions from version file ? Basically same version for ccp & overlay, so what we follow there, just do the same here?

Reply:
as discussed, we will be updating this to pickup the version value from values.yaml i.e. the actual version of the agent being deployed instead of this static value and this is going to be in a separate PR.